config/reference_paths.config

```
@@ -7,5 +7,6 @@ ref_rootdir  = 's3://scpca-references'
 barcode_dir  = "${params.ref_rootdir}/barcodes/10X"
 
 // cell type references
-celltype_ref_dir = "${params.ref_rootdir}/celltype/references" // original (generally celldex) files themselves
-celltype_model_dir = "${params.ref_rootdir}/celltype/singler_models" // models build from original (celldex) files
+celltype_ref_dir = "${params.ref_rootdir}/celltype"
+celltype_ref_metadata = "${projectDir}/references/celltype-reference-metadata.tsv"
+panglao_marker_genes = "${projectDir}/references/PanglaoDB_markers_27_Mar_2020.tsv"
```

```suggestion
panglao_marker_genes_file = "${projectDir}/references/PanglaoDB_markers_27_Mar_2020.tsv"
```

to confirm, this is just getting set up for next steps and isn't used yet, seems fine! Suggested name change just to be clear it's a _file_ and not, for example, straight-up a list of genes.